kubernetes_deploy/scripts/job.sh

```diff
@@ -3,8 +3,8 @@ function _job() {
   usage="job -- run job in the specified environment
```

Comment:
context='live-1' (line 45) fails for me (not something introduced by this change though) - should be context='live-1.cloud-platform.service.justice.gov.uk' to make it generic for all devs?

Reply:
@kmahern I have modified to use the users `--current` context and just changes `--namespace` - can you give the job a go again and let me know if that fixes the issues

Reply:
👍 yes this worked.